content/en/monitors/guide/slo-checklist.md

```diff
@@ -0,0 +1,107 @@
+---
+title: SLO checklist
+kind: guide
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/slo-monitoring-tracking/'
+      tag: 'Blog'
+      text: 'Track the status of your SLOs with the new monitor uptime widget'
```

Comment:
Monitor uptime widget was deprecated a long time ago. Let's just change this to "Track the status and error budget of your SLOs with Datadog"